### DISCUSSION TITLE
How to access component functions from inside nested object

### DISCUSSION THREAD

**Discussion by Alexhandru** [rsuite/rsuite#1593] [Q&A]
Hi guys, in the example https://github.com/rsuite/rsuite/blob/master/docs/pages/components/table/en-US/default.md, I was wondering how can one for example call a function of FixedColumnTable inside the nested object rowData?

**Comment by simonguo** [comment]
I don't understand your question, can you describe it in detail?

**Reply by Alexhandru** [reply]
So what I actually am trying to achieve is, how do I pass rowData to a function of FixedColumnTable? Like I would need the information in rowData to also be available in FixedColumnTable.
